pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -244,6 +244,22 @@ def test_tick_division(cls):
         assert result.delta == off.delta / 0.001
 
 
+def test_tick_mul_float():
+    off = Micro(2)
+
+    # Case where we retain type
+    result = off * 1.5
+    expected = Micro(3)
+    assert result == expected
+    assert isinstance(result, Micro)
+
+    # Case where we bump up to the next type
+    result = off * 1.25
```

Comment:
was this buggy before? need a whatsnew note if so

Reply:
ATM it raises TypeError for any float, will add whatsnew